bpf_test.go

```diff
@@ -255,7 +255,7 @@ func TestModuleLoadELF(t *testing.T) {
 		t.Fatal("prog is nil")
 	}
 	if err := b.Load(nil); err != nil {
-		if !strings.Contains(err.Error(), "invalid argument") {
```

Comment:
I wonder why we ignore the error at all? @iaguis do you remember? Changed with 8f18c857026f0495eed5b00c28f9697548d88519

Reply:
I don't really know :/

I think we can just remove it.

Reply:
@iaguis Removed the check for invalid argument.